README.md

```diff
@@ -18,11 +18,13 @@ aspen% . .venv/bin/activate
 2. Install pipenv.
 ```bash
 (.venv) aspen% pip install pipenv
-(.venv) aspen%
+(.venv) aspen% 
 ```
 3. Install the packages required for development.
 ```bash
 (.venv) aspen% pipenv install --dev
+(.venv) aspen% pipenv shell
```

Comment:
this shouldn't be necessary.

Reply:
I just tested it and it's not required.

Reply:
it was for me..

Reply:
because when you do `pipenv install --dev` it installs all of the dev packages into a virtual env that you then can't access until you do `pipenv shell`. I'm using conda as my base environment (where really just `pipenv` is installed..). I wonder if using virtualenv is more seemless.

Reply:
@lvreynoso did you have to run pipenv shell to access the project packages? Are you using virtualenv too?

Reply:
When I did it, `pipenv` said that it detected an existing virtualenv and will use it.  I'm guessing that's the difference.  Are you creating a virtualenv from conda?

Reply:
I had the same experience as @ttung since I am using a `pyenv virtualenv` that I created for aspen using python 3.7.9. So I didn't actually have to do Step 1 of the Readme, or worry about doing `pipenv shell`. 

But if that's what you have to do when following our instructions exactly then it should be included :)

Reply:
Ok I verified that doing `pipenv shell` is harmless in the virtualenv way.